core/modules/loaders.rs

```diff
@@ -293,8 +304,13 @@ impl ModuleLoader for FsModuleLoader {
         ModuleType::JavaScript
       };
 
-      let code = std::fs::read_to_string(path)?.into();
-      let module = ModuleSource::new(module_type, code, module_specifier);
+      // TODO(bartlomieju): handle bytes as well
+      let code: FastString = std::fs::read_to_string(path)?.into();
+      let module = ModuleSource::new(
+        module_type,
+        ModuleSourceCode::String(code),
+        module_specifier,
+      );
```

Comment:
In here I'm actually erring on the side of always returning bytes - since we have a helper that tries to convert provided source code to a string this should be all fine (and would handle other types of modules like WASM and bytes easily. TBD